src/coreclr/jit/emitxarch.cpp

```diff
@@ -6847,10 +6898,10 @@ void emitter::emitIns_R_R_A(
     id->idIns(ins);
     id->idReg1(reg1);
     id->idReg2(reg2);
-    if (instOptions == INS_OPTS_EVEX_b)
+    if (instOptions == INS_OPTS_EVEX_eb_er_rn)
```

Comment:
Does this need to be `(instOptions & SomeMask) == INS_OPTS_EVEX_eb_er_rn` (or some helper to extract the relevant bits, etc)

Reply:
-- Just thinking from the perspective that `options` generally implies more than one can be set, so it may be relevant to add masking now so it isn't an issue later